discord/errors.py

```diff
@@ -213,3 +214,9 @@ def __init__(self, shard_id):
               'and explicitly enable the privileged intents within your application\'s page. If this is not ' \
               'possible, then consider disabling the privileged intents instead.'
         super().__init__(msg % shard_id)
+
+
+class RecordingException(ClientException):
+    """Exception that's thrown when there is an error while trying to record
+    audio from a voice channel."""
```

Comment:
See [PEP 8's Documentation Strings](https://www.python.org/dev/peps/pep-0008/#documentation-strings). The ending quotes should be on a third line in this case. Unless it just fits on one line?

Reply:
Ohhh I see, thanks for pointing that out.